PWGLF/TableProducer/lithium4analysis.cxx

```diff
@@ -384,9 +437,23 @@ struct lithium4analysis {
                 continue;
               }
 
-              if (!FillCandidateInfo(track1, track2, false)) {
+              if (!FillCandidateInfo(track1, track2, false, true)) {
                 continue;
               }
+
+              // fill TOF info outside to avoide responseBeta crash
+              auto& cand = l4Candidates.back();
+              if (track1.hasTOF()) {
+                float beta = responseBetaMC.GetBeta(track1);
```

Comment:
Ciao Francesco, thanks a lot! Is this because you want to avoid computing beta for all tracks?

Reply:
Ciao Nicolò, yes!

Reply:
maybe we should make it a dynamic column, thx!